app/assets/stylesheets/barnardos/_textarea.scss

```diff
@@ -62,6 +62,9 @@
 .textarea__hint {
   color: $secondary-text;
   display: block;
+  font-family: $font-family;
```

Comment:
Ok. I've made a *lot* of markup in gallery, and put some gash styling in the top so we can see what code applies to which examples. This covers things like hints.

Are there any CSS unit testing frameworks we should be considering? 😛